src/components/molecules/NoughtsAndCrosses/NoughtsAndCrosses.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from "react";
+import "./NoughtsAndCrosses.css";
+
+export default function NoughtsAndCrosses() {
+  const [gameState, setGameState] = useState([
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+  ]);
+  const [currentPlayer, setCurrentPlayer] = useState("X");
+  const [winner, setWinner] = useState(null);
+
+  function playTurn(squareIndex) {
+    if (winner === null) {
+      if (gameState[squareIndex] === null) {
+        var newGameState = [...gameState];
```

Comment:
Do you think we should add a comment saying what the triple ...  does?

Reply:
Good point, should add comments to explain different sections

Reply:
Added comments